.goreleaser.yml

```diff
@@ -0,0 +1,104 @@
+before:
+  hooks:
+    - go mod tidy
+    - go test ./...
+builds:
+  - id: fw-id-agent
+    binary: fw-id-agent
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+    main: ./cmd/fw-id-agent/main.go
+    ldflags:
+      - -s -w -X github.com/T-Systems-MMS/fw-id-agent/internal/agent.Version={{.Version}}-{{.Commit}}
+  - id: fw-id-cli
+    binary: fw-id-cli
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+    main: ./cmd/fw-id-cli/main.go
+    ldflags:
+      - -s -w -X github.com/T-Systems-MMS/fw-id-agent/internal/agent.Version={{.Version}}-{{.Commit}}
+archives:
+  - format: tar.gz
+    rlcp: true
+    wrap_in_directory: true
+    files:
+      - src: init/fw-id-agent.service
+        dst: fw-id-agent.service
+        info:
+          mode: 0644
+      - src: configs/config.json
+        dst: example_config.json
+        info:
+          mode: 0644
+      - README.md
+      - LICENSE
+checksum:
+  name_template: "checksums.txt"
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+nfpms:
+  - package_name: fw-id-agent
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
+    vendor: Deutsche Telekom MMS GmbH
+    maintainer: The MMS Linux Dev Team <mmslinux-dev@t-systems-mms.com>
+    description: |-
+      Corporate Identity Agent
+      Agent to logon at the corporate firewall.
```

Comment:
I would prefer the name "Firewall Identity Agent" so it matches the title we have in the README.
Some alternative suggestions for the description: "Agent to log in on the corporate firewall.", "Agent for logging in on the corporate firewall." "...log on to...", "... logging on to..."